javatests/arcs/core/entity/EntityBaseTest.kt

```diff
@@ -27,11 +28,17 @@ import org.junit.runners.JUnit4
 @RunWith(JUnit4::class)
 @Suppress("UNCHECKED_CAST")
 class EntityBaseTest {
-    private lateinit var entity: EntityBaseForTest
+    private lateinit var entity: DummyEntity
 
     @Before
     fun setUp() {
-        entity = EntityBaseForTest()
+        SchemaRegistry.register(DummyEntity)
```

Comment:
Might be better to add an init {} block with auto-registration to the DummyEntity to better simulate the way it will happen for the developer, just to catch any lazy initialization issues.

Reply:
I think I prefer to make it explicit in the test, it's easier to understand what's going on.